client/src/Pages/Infrastructure/Create/index.jsx

```
@@ -96,6 +112,35 @@ const CreateInfrastructureMonitor = () => {
+			usage_temperature,
+			...rest
+		} = form;
+
+		const thresholds = {
+			...(cpu ? { usage_cpu: usage_cpu / 100 } : {}),
+			...(memory ? { usage_memory: usage_memory / 100 } : {}),
+			...(disk ? { usage_disk: usage_disk / 100 } : {}),
+			...(temperature ? { usage_temperature: usage_temperature / 100 } : {}),
+		};
+
+		form = {
```

Biome (2.1.2)

[error] Can't assign form because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)